### DISCUSSION TITLE
Rated 1v1 should not give arena points and just flat out removed

### DISCUSSION THREAD

**Discussion by Never6** [chromiecraft/chromiecraft#7436] [Suggestions]
title says it all, you should not be rewarded with high end arena gear because your class has a better toolkit at dueling that others
unrated is fine cause that is more or less what you can do outside org for fun, but giving rewards is a bad idea

**Comment by Apathyxia** [comment]
I'm not active on the server anymore, but I do have some opinions regarding this, and perhaps this could ignite some motivation for my return.

I doubt people will play unrated 1v1 arenas that much. Only those that generally duel outside Stormwind or Orgrimmar. Including some form of rewards would make that more active too. I'm not saying the arena point gain should be similar to 2s or 3s. What I have evidently seen when I was more active, even in "rock paper scissors" match ups, people playing rock against scissors would lose 75% of the time. The skill levels on this server are not high. Applying your suggested logic to 2s: it should not have rewards because some comps are clearly better than others. It does not make sense.

What I see is that 1v1 rated arena could incentivise people to also queue during off-peak hours when there normally aren't people queueing for 2s or even BGs. So many EU mornings when I still played I was faced with 3 or 4 out of 6 people in WSG queue. How much nicer it would be to now just queue 1v1s.

1v1 has some other problems. One being that healers should not be able to queue in this game mode, because what would be more fun than a resto druid vs. a resto druid. I don't think healers are duelling that much in the first place. 
Other problem, especially with introduced rewards, is that wintrading would become a lot easier. This would require more monitoring over what is happening in the arena. 

-Apatia

**Reply by Never6** [reply]
I agree healers shouldn't be able to queue
i will straight up leave and take DC penalty over 20min resto 1v1
also the way the announcement  is worded makes it sound like you get points just for playing which I think is also wrong

**Reply by Apathyxia** [reply]
I think it was not meant that way, but it clearly was written that way. There should not be an incentive to not play. Then again, if a person dies in a rogue stun lock, I'm not sure if that can be counted as "not playing". :D

**Comment by Helias** [comment]
Thanks for the suggestions. I am trying to collect as much feedback as possible on this.

> Other problem, especially with introduced rewards, is that wintrading would become a lot easier. This would require more monitoring over what is happening in the arena.

We agree that, as you mentioned, 1v1 arenas are an easy target for win-traders. In fact, if we enable arena point rewards, we are considering setting a significantly lower reward compared to other arena formats.

Personally, I also think that 3v3 solo queue should give fewer arena points than standard 2v2 or 3v3 matches.

As we mentioned in the article, for now, 1v1, 3v3 solo queue, and other new features like arena replay and NPC spectators are in beta. While they remain in beta, no rewards will be given. We would like to take more time to evaluate them, including considering your feedback, so thanks for starting this discussion.

**Reply by lamsxx** [reply]
I don't know how this has been implemented, but it's certain that heal classes are not allowed to access 1vs1 arenas. Each class must use its DPS spec and not its heal spec; otherwise, it won't be balanced and it will lead to problems if, for example, two heal classes fight each other or if a heal faces a DPS. Regarding the rewards, I agree with the idea of slightly decreasing the number of arena points earned each week in 1vs1, as it requires less effort than 2vs2 and is more accessible (no need to find a partner).

**Reply by Apathyxia** [reply]
I actually think 3v3 solo queue should not give less arena points. The possibility of win trading is lower in that setting than in normal matches since you have more or less randomised teams. 

After all, solo queue takes a lot less effort to queue into than finding a full 3s team, and on top that a second 3s team to queue against. I am sure there are players that would like to play more casually, too. 

When it comes to 3v3, an important consideration is to try to balance the teams somehow. By this I mean that the system should try to ensure that both teams have the same amount of healers. It should not be a possible outcome to have 2 healers + dps vs. 3 dps.

How to do this? I am not sure. An idea I have is the following:
- Define each spec as dps or healer (tanks I guess go into dps category)
- When a person queues into solo queue, their current spec defines their role they queue in. For example, if they have most talent points in restoration druid spec, they would be queueing as healer. 
- - They would be blocked from changing specs while in the queue. 
- - Would have to make sure that, say, person starts changing specs, then queues, it would not be able to happen. I.e., if a person changes specs, it drops queue. Or if a match pops while changing spec, it declines the queue. Or something else to counter any misplay attempts with changing specs.

**Comment by Helias** [comment]
> Rated 1v1 should not give arena points

As we announced it on discord, we are giving really a small quantity of ArenaPoints.

> I don't know how this has been implemented, but it's certain that heal classes are not allowed to access 1vs1 arenas.

There are some filters but not what you mean, we could improve it in the future for sure.

> I actually think 3v3 solo queue should not give less arena points. 

the 3v3 arena points reward multiplier is `0.88` the 3v3soloQ is quite the same `0.8`, all the details about the AP distribution is present in the ArenaStats, clicking on "Arena Points distribution" it's possible to see the related formula and multiplier  https://www.chromiecraft.com/en/arena-stats/

Thanks for all the suggestions, all the feedback helped us to make a balanced and thoughtful decision on these topics.
I am closing this discussion.
